tests/lib/local_repos.py

```diff
@@ -8,6 +8,7 @@
 from pip._internal.utils.misc import hide_url
 from pip._internal.vcs import bazaar, git, mercurial, subversion
 from tests.lib import path_to_url
+from tests.lib.path import Path
```

Comment:
Guarded with `MYPY_CHECK_RUNNING`? I see we only do this in `tests/lib/__init__.py` so maybe it's not a big deal.

Reply:
Sure, I can make that change. Thanks. (I feel like our linters should be catching this..)